lib/locale/pl_PL/strings.php

```diff
@@ -4984,6 +4986,10 @@
 $_LANG['Negative value means customers with karma less or equal than specified, but positive means greater or equal then specified'] = 'Wartość ujemna oznacz wyszukiwanie klientów z karmą mniejszą lub równą od podanej, zaś dodatna większą lub równą od podanej';
 
 $_LANG['Customer notifications'] = 'Powiadomienia klienta';
+$_LANG['New ticket customer notification subject'] = 'Temat powiadomienia klienta o nowym zgłoszeniu';
+$_LANG['New ticket customer notification body'] = 'Treść powiadomienia klienta o nowym zgłoszeniu';
+$_LANG['New message customer notification subject'] = 'Temat powiadomienia klienta o nowej wiadomości';
+$_LANG['New message customer notification body'] = 'Treść powiadomienia klienta o nowej wiadomości';
 $_LANG['Verifier notifications'] = 'Powiadomienia weryfikatora';
```

Comment:
Skąd to się tutaj wzięło? Jakieś zagubione tłumaczenia, które również powinny trafić do **stable**?

Reply:
Tak - napisałem to wyżej.

Reply:
Czyli tym razem robię merge bez squasha?

Reply:
Dokładnie no chyba, że chcesz żebym to jakoś pokroił w PR ale uważam, że nie ma sensu bo są to mega małe zmiany.